tern/classes/package.py

```diff
@@ -12,7 +12,7 @@ class Package:
     attributes:
         name: package name
         version: package version
-        license: package license
+        license : package license
```

Comment:
This change is not needed

Reply:
Or rather, if you want to follow the same pattern throughout the file, it should be `pkg_license: package license`

Reply:
Sure will do the change!